keylime/registrar_client.py

```diff
@@ -64,7 +68,20 @@ def getData(registrar_ip, registrar_port, agent_id, tls_context):
             logger.critical("Error: did not receive port from Registrar Server.")
             return None
 
-        return response_body["results"]
+        r = response_body["results"]
+        res: tenant.RegistrarData = {
+            "aik_tpm": r["aik_tpm"],
+            "regcount": r["regcount"],
+            "ek_tpm": r["ek_tpm"],
+            "ip": r["ip"],
+            "port": r["port"],
+            "mtls_cert": r.get("mtls_cert"),
+            "ekcert": r.get("ekcert"),
```

Comment:
From looking through tenant.py "ekcert" should always be there but may be None. However, this one is not checked in this function above.